### DISCUSSION TITLE
How to setup GitLens in VSCodium?

### DISCUSSION THREAD

**Discussion by ywiyogo** [VSCodium/vscodium#1695] [General]
Hi everyone,
I've installed the vsix of GitLens. However, on the left bar of GitLens, I can't see the commit list, instead it shows:
```
There is no data provider registered that can provide view data.
```
I also see a popup error:
```
Error running command gitlens.views.commits.refresh: command 'gitlens.views.commits.refresh' not found. This is likely caused by the extension that contributes gitlens.views.commits.refresh.
```
What should I do to see the commit list of my working directory?

Thanks in advance.

**Comment by daiyam** [comment]
`command 'gitlens.views.commits.refresh' not found.` is an error when an extension isn't correctly installed.

Just tested, no issue for me.

**Reply by ywiyogo** [reply]
Thanks for your verification! I downloaded the latest version eamodio.gitlens-2023.10.1405.vsix and reinstall the extension. It works now. The previous version I installed that was broken, was eamodio.gitlens-2023.9.2512.vsix.
